egs/wsj/s5/steps/nnet3/chain/build_tree.sh

```diff
@@ -22,9 +22,6 @@ cmd=run.pl
 context_opts=  # e.g. set this to "--context-width 5 --central-position 2" for quinphone.
 cluster_thresh=-1  # for build-tree control final bottom-up clustering of leaves
 frame_subsampling_factor=1
-leftmost_questions_truncate=-1  # note: this used to default to 10, but we never
-                                # use this option now with value != -1, and
-                                # we're changing the default
 tree_stats_opts=
```

Comment:
I guess you shouldn't remove it from here so that the old scripts calling this script won't crash.